elasticsearch/src/main/java/com/gentics/mesh/search/ElasticsearchProcessManager.java

```diff
@@ -206,7 +206,8 @@ private String getJavaBinPath() throws FileNotFoundException {
 	private File prepareESDirectory() throws IOException, ZipException {
 		File outputDir = new File(".");
 		File esDir = new File(outputDir, OUTPUT_FOLDER_NAME);
-		if (!esDir.exists()) {
+		File esBin = new File(esDir, "bin");
+		if (!esBin.exists()) {
 			unzip("/" + RESOURCE_NAME + ".zip", outputDir.getAbsolutePath());
```

Comment:
This can potentially take a long time now. Could this run in any timeouts on slow IO systems?

Reply:
I don't think this is relevant since we will drop ES embedded support in the future. The default mode for embedded ES is already set to false.